evm/src/libraries/TrimmedAmount.sol

```diff
@@ -105,6 +105,31 @@ library TrimmedAmountLib {
         return TrimmedAmount(a.amount + b.amount, a.decimals);
     }
 
+    function saturatingAdd(
+        TrimmedAmount memory a,
+        TrimmedAmount memory b
+    ) internal pure returns (TrimmedAmount memory) {
+        // on initialization
+        if (isZero(a)) {
```

Comment:
we should do something about these (separately). I feel like these checks should be somewhere on the initialisation path

Reply:
Created https://github.com/wormhole-foundation/example-native-token-transfers/issues/222